### ISSUE TITLE
logit_bias is Defined but Never Used in generate

### ISSUE THREAD

**Issue by nmehran** [Zyphra/Zonos#77]
In the `generate` function, the `logit_bias` tensor is created to prevent all but the first codebook from predicting `eos_token_id`, but it is never applied to the logits before sampling. This means it has no effect on the generation process.

   ```python
   logit_bias = torch.zeros_like(logits)
   logit_bias[:, 1:, self.eos_token_id] = -torch.inf
   ```
However, it is never added to `logits` before calling `sample_from_logits`.

```python
logits = self._decode_one_token(input_ids, inference_params, cfg_scale)
logits += logit_bias  # Apply bias before sampling ?
next_token = sample_from_logits(logits, generated_tokens=delayed_codes[..., :offset], **sampling_params)
```

**Comment by sekstini** [comment]
Yup, you're completely right! Fixed in https://github.com/Zyphra/Zonos/commit/467de6bd7704cd9b63af2082b4819c88b74deffe
